src/javascripts/ticket_sidebar.js

```diff
@@ -1,18 +1,41 @@
 import View from 'view';
-import Storage from 'storage'
+import Storage from 'storage';
+import I18n from 'i18n';
 
-const MAX_HEIGHT = 375;
+const maxHeight = 375;
 
 class TicketSidebar {
   constructor(client, data) {
     this.client = client;
     this._metadata = data.metadata;
     this._context = data.context;
-
     this.storage = new Storage(this._metadata.installationId);
+
+    // Get user function
+    const getUser = this.getCurrentUser();
+    
+    // Resize function
+    const resizeApp = (appHeight, appWidth) => client.invoke('resize', { height: appHeight, width: appWidth });
+
+    // Translate function
+    const translate = (keyword) => I18n.t(keyword);
+
     this.view = new View({ afterRender: () => {
```

Comment:
We could translated prior to `afterRender` is called then append the message after?